indigo_content_api/v1/views.py

```diff
@@ -395,3 +395,9 @@ def determine_place(self):
             raise Http404
 
         super(PublishedDocumentSearchView, self).determine_place()
+
+
+class TaxonomyView(viewsets.ReadOnlyModelViewSet):
+
+    queryset = TaxonomyVocabulary.objects.all()
```

Comment:
Putting `.all()` at the end causes the query to be loaded from the database immediately (when the class is declared), which means that it will never reflect updates in the database. You can simply set the queryset without the `.all()`.